extensions/amp-3d-gltf/0.1/amp-3d-gltf.js

```diff
@@ -96,7 +96,7 @@ export class Amp3dGltf extends AMP.BaseElement {
         this.element);
 
     this.context_ = dict({
-      'src': src,
+      'src': resolveRelativeUrl(src, this.win.location),
```

Comment:
please use `this.getAmpdoc().getUrl()` instead of `this.win.location`

Reply:
done